fitbenchmarking/controllers/minuit_controller.py

```diff
@@ -5,6 +5,7 @@
 http://iminuit.readthedocs.org
 """
 from iminuit import Minuit
+import iminuit
```

Comment:
I think we should only use 1 import style, we should either import just `Minuit` and `__version__` as iminuit_version or something sensible, or import iminuit and use `iminuit.Minuit`